Habitica/src/main/java/com/habitrpg/android/habitica/ui/fragments/BaseFragment.java

```diff
@@ -133,5 +108,32 @@ public String getDisplayedClassName() {
         return this.getClass().getSimpleName();
     }
 
+    private class TutorialStepTransactionListener implements TransactionListener<TutorialStep> {
```

Comment:
it's the static inner classes that keep a reference to the outer class if they remain after the activity goes away